ososedki_dl/crawlers/other/wildskirts.py

```
@@ -5,22 +5,26 @@
 
     from bs4 import BeautifulSoup
-    from bs4.element import NavigableString, Tag
+    from bs4.element import Tag
 
 
 class WildskirtsCrawler(BaseCrawler):
     site_url = "https://wildskirts.com"
     base_photos_url: str = "https://photos.wildskirts.com"
     base_videos_url: str = "https://video.wildskirts.com"
+    api_url: str = site_url + "/api/media"
+    headers = {"Referer": site_url + "/"}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)